### PR TITLE
Build system refactor

### PR DESCRIPTION
**Pull request by NicerNewerCar**

### CODE REVIEW

File: autoscoper/CMakeLists.txt [hunk 1]
```diff
@@ -1,171 +1,171 @@
-SET(CMAKE_CURRENT_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src)
```
Comment (jcfr) [inline autoscoper/CMakeLists.txt:1]: Look like the line endings of text file is inconsistent (dos vs unix), could you check that his is why the complete file ends up being modified ?

**Comment by jcfr** [comment]
Looks great overall :100: 

Nitpick: In commit message title, consider using imperative mood. `Remove` instead of `Removed`. 

Few references:
* https://slicer.readthedocs.io/en/latest/developer_guide/style_guide.html#commits
* https://cbea.ms/git-commit/

File: autoscoper/CMakeLists.txt (hunk 1 above)
Comment (NicerNewerCar) [reply autoscoper/CMakeLists.txt:1]: Yeah that looks like what is happening, I can try updating the script and rebasing the code

Comment (NicerNewerCar) [reply autoscoper/CMakeLists.txt:1]: @jcfr I'm not sure what happening as this only happened in the one files for the [Convert to lower case commit](https://github.com/BrownBiomechanics/Autoscoper/pull/10/commits/b3597627629d33cc3dbed21aaed300a23f49ab15) so I don't think it has to do with the line endings.

Comment (jcfr) [reply autoscoper/CMakeLists.txt:1]: Thanks for looking into it, I just checked out the branch locally and will follow up in a few minutes.

**Pull request merged**